bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fineoffsetweatherstation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="gateway">
+		<label>Gateway Device</label>
+		<category>NetworkAppliance</category>
+		<config-description-ref uri="thing-type:fineoffsetweatherstation:gateway"/>
+	</bridge-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wind-speed">
```

Comment:
Can system channel `system.wind-speed` be used instead?

Reply:
I already use `system.wind-speed` https://github.com/openhab/openhab-addons/pull/12464/files#diff-f38f93267a029e6a7f62dddaa4ccf3fbf2c9dfba61832d8d0f83662343afdfddR62-R63

Reply:
If `system.wind-speed `is used, is this custom `wind-speed` then unused and can be removed?

Reply:
So there are 3 different channels regarding wind speed:

* speed-wind
* speed-gust
* wind-max-day

Since the latter one is a statistical one, I would rename the channel type to `max-wind-speed` and remove the tags as well.